libs/pageserver_api/src/shard.rs

```diff
@@ -18,6 +18,20 @@ impl ShardNumber {
     pub const MAX: Self = Self(u8::MAX);
 }
 
+/// TenantShardId identify the units of work for the Pageserver.
+///
+/// Historically, tenants only had one shard, and were identified
```

Comment:
We still support the 1-shard case, right? Maybe it should read "Historically, tenants always had one shard"?

Reply:
I've fleshed this comment out a bit.